client/src/components/MapLayout/index.jsx

```diff
@@ -9,70 +9,100 @@ import SelectCategory from '../EventForm/SelectCategory';
 class MapLayout extends Component {
   state = {
     events: [],
-    day_filter: 0,
+    date_filter: '0',
     category: ''
   }
 
+  onChangeHandlerRadio = (e) => {
+    this.setState({ date_filter: e.currentTarget.value })
+  };
+
   onChangeHandler = (e) => {
-      let {id} = e.currentTarget
-      this.setState({[id]: e.currentTarget.value})
+    let { id } = e.currentTarget
+    this.setState({ [id]: e.currentTarget.value })
   };
 
-  filter = (e) => {
-      let filterData = {
-          day_filter: this.state.day_filter,
-          category: this.state.category
+  filter = () => {
+    let filterData = {
+      day_filter: this.state.date_filter,
+      category: this.state.category
+    }
+    request('/events-short-info', 'POST', JSON.stringify(filterData)).then(data => {
+      if ('info' in data) {
+        this.setState({ events: data.info })
       }
-      request('/events-short-info', 'POST', JSON.stringify(filterData)).then(data => {
-          if ('info' in data) {
-              this.setState({ events: data.info })
-          }
-      });
+    });
   }
 
-  componentDidMount() {
-      let filterData = {
-          day_filter: this.state.day_filter
+  componentDidMount = () => this.filter();
+    renderAddEventButton() {
+      if (isLogged()) {
+        return (
+          <div className="fixed-action-btn horizontal">
+            <Link className="btn-floating btn-large red" to="/add-event">
+              <Icon>add</Icon>
+            </Link>
+          </div>
+        )
       }
-      request('/events-short-info', 'POST', JSON.stringify(filterData)).then(data => {
-          if ('info' in data) {
-              this.setState({ events: data.info })
-          }
-      });
-  }
-
-  renderAddEventButton() {
-    if (isLogged()) {
-      return (
-        <div className="fixed-action-btn horizontal">
-          <Link className="btn-floating btn-large red" to="/add-event">
-            <Icon>add</Icon>
-          </Link>
-        </div>
-      )
     }
-  }
 
-  render(){
+  render() {
     return (
       <Row>
-          <Input id='day_filter' s={3} type='select' label="Filter by days" defaultValue='' onChange={this.onChangeHandler}>
-              <option value="" disabled selected>Choose filter</option>
-              <option value='1' >Today</option>
-              <option value='7'>One week</option>
-              <option value='14'>Two weeks</option>
-          </Input>
-          <Button onClick={this.filter}>Filter</Button>
-          <Col s={3}>
-              <SelectCategory onChangeHandler={this.onChangeHandler}/>
-          </Col>
+        <Col>
+          <p>
+            <input
+              className="with-gap"
+              name="group1"
```

Comment:
I don't like numbers in names, I think better call like group_one or group_one_day, and same with id names.

Reply:
Can you argue?